src/public/index.html

```diff
@@ -5,6 +5,7 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0">
   <link href="assets/favicon.ico" rel="icon" type="image/x-icon">
+  <link rel="stylesheet" href="../../node_modules/material-design-lite/material.min.css">
```

Comment:
You should be using a `require` statement for the css so it's bundled. Otherwise this won't work when deploying. Look at webpack.base.config to add the css extension last and add the require to app.ts. you should be able to require material-design-lite/material.min!